core/src/test/java/org/pentaho/di/core/database/RemedyActionRequestSystemDatabaseMetaTest.java

```diff
@@ -34,12 +34,11 @@ public class RemedyActionRequestSystemDatabaseMetaTest {
   @Before
   public void setupBefore() {
     odbcMeta = new RemedyActionRequestSystemDatabaseMeta();
-    odbcMeta.setAccessType( DatabaseMeta.TYPE_ACCESS_ODBC );
   }
 
   @Test
   public void testSettings() throws Exception {
-    assertArrayEquals( new int[] { DatabaseMeta.TYPE_ACCESS_ODBC, DatabaseMeta.TYPE_ACCESS_JNDI },
+    assertArrayEquals( new int[] { DatabaseMeta.TYPE_ACCESS_JNDI },

```

Comment:
Check review for "RemedyActionRequestSystemDatabaseMeta.java" class: this test class should be marked as deprecated and later removed.